mk/uvmt/README.md

```diff
@@ -15,20 +15,33 @@ To run the UVM environment you will need:
 - the RISC-V GCC compiler, and
 - at least some familiarity with [Make](https://www.gnu.org/software/make/manual/) and Makefiles.
 
-COREV Environment Variables
+Required COREV Environment Variables
+------------------------------------
+The following environment variables **must** be set for any `make` invocation to run tests or set in the user's environment to customize setting.
+
+| Environment Variable | Description  |
+|----------------------|--------------|
+| CV_SIMULATOR         | The default simulator to use for all tools (dsim, vcs, xrun, vsim, riviera).  Can be overridden on any make invocation. |
+| CV_CORE              | The core to simulate by default.  Can be overridden on any make command line.  Typically specified in the same case as the directory in which the testbench resides (e.g. cv32e40p, cva6).  However makefiles have access to internally defined CV_CORE_LC and CV_CORE_UC variables to get expected cases of the CV_CORE name (e.g. for macros, etc that might typically be capitalized). |
+| CV_SW_TOOLCHAIN      | Points to SW toolchain installation for compiling, assembling, and/or linking test programs. |
+| CV_SW_PREFIX         | Prefix of the SW toolchain installation. |
+| CV_SW_MARCH          | Architecture of tool chain to invoke. The default is `rv32imc`. |
```

Comment:
Since this has a default, could it be in "Optional" instead of in "Required"?

Reply:
I would prefer the defaults to be set in the cfg-yaml files; a user should not need to look into the makefiles themselves unless strictly necessary, thus dropping the default from the makefile, and make it a required argument makes sense to me

Reply:
I looked upon this as a list of user-specified variables.
So "required" means "you must set up your environment like this or else nothing will run".
While "optional" means "you may tweak, but defaults already exist".

But I agree that the cfg-yaml can set CFG_COREV_MARCH, and that RISCV_MARCH maybe doesn't need any default in the makefile itself.

Reply:
@silabs-hfegran, that is a good idea. I will work on that and push in an update to this PR.

@silabs-robin, you have it correct.  I will steal your wording and add it to the README.